roles/edpm_network_config/defaults/main.yml

```diff
@@ -30,15 +30,12 @@ edpm_network_config_tool: os-net-config
 edpm_network_config_update: false
 edpm_network_config_async_poll: 3
 edpm_network_config_async_timeout: 300
-edpm_network_config_bridge_name: br-ex
```

Comment:
could it be that we should rename `neutron_physical_bridge_name` ?

Reply:
I would argue that it's easier to rename it here, in one place, rather than everywhere else. The `neutron_physical_bridge_name` also seems more descriptive.

Reply:
yeah but it would be better to have the `edpm_network_config_` prefix just like the other vars

Reply:
I agree that we should put this PR on hold, and i'm surprised to see that the var is widely used but it's not documented and doesn't have a "default" in the role where it should be used.